modules/utxo-lib/src/noble_crypto.ts

```diff
@@ -0,0 +1,76 @@
+import * as createHash from 'create-hash';
```

Comment:
Is there a good reason not to combine this with `noble_ecc` and have one file that exports all of these related cryptographic primitives?

Reply:
No, no strong opinions here on this. Updating it so that it's combined with noble_ecc